src/api/app/views/webui2/webui/projects/meta/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,24 @@
+= render partial: 'webui/project/breadcrumb_items'
```

Comment:
The meta part should still be here though.

Reply:
![meta3](https://user-images.githubusercontent.com/37418/49371644-7892a980-f6f8-11e8-9862-fb9ca0537051.png)

it is there, no?

Reply:
I meant that it should be in this file, instead of being in: https://github.com/openSUSE/open-build-service/blob/b8a5640a4aecc95142e95803ff885b92e85cb7d9/src/api/app/views/webui2/webui/project/_breadcrumb_items.html.haml#L22-L24